ligo/test/Ligo/BaseDAO/Types.hs

```diff
@@ -0,0 +1,274 @@
+    T.Set_quorum_threshold p ->
+      Set_quorum_threshold p
+    T.Set_voting_period p ->
+      Set_voting_period p
+    T.Transfer_contract_tokens p ->
+      Transfer_contract_tokens p
+    T.Transfer_ownership p ->
+      Transfer_ownership p
+    T.Vote p ->
+      Vote p
+
+  fromLigo = \case
+    Accept_ownership p ->
+      T.Accept_ownership p
+    Burn p ->
+      T.Burn p
+    CallCustom p ->
+      T.CallCustom p
+    Call_FA2 p ->
+      T.Call_FA2 p
+    Confirm_migration p ->
+      T.Confirm_migration p
+    Drop_proposal p ->
+      T.Drop_proposal p
+    Flush p ->
+      T.Flush p
+    GetVotePermitCounter p ->
+      T.GetVotePermitCounter p
+    Migrate p ->
+      T.Migrate p
+    Mint p ->
+      T.Mint p
+    Propose p ->
+      T.Propose p
+    Set_quorum_threshold p ->
+      T.Set_quorum_threshold p
+    Set_voting_period p ->
+      T.Set_voting_period p
+    Transfer_contract_tokens p ->
+      T.Transfer_contract_tokens p
+    Transfer_ownership p ->
+      T.Transfer_ownership p
+    Vote p ->
+      T.Vote p
+
+data Storage = Storage
+  { sAdmin :: Address
+  , sExtra :: ContractExtraL
+  , sLedger :: Ledger
+  , sMetadata :: "metadata" :! TZIP16.MetadataMap BigMap
+  , sMigrationStatus :: MigrationStatus
+  , sOperators :: Operators
+  , sPendingOwner :: Address
+  , sPermitsCounter :: Nonce
+  , sProposals :: BigMap ProposalKeyL ProposalL
+  , sProposalKeyListSortByDate :: Set (Timestamp, ProposalKeyL)
+  , sQuorumThreshold :: QuorumThreshold
+  , sTokenAddress :: Address
+  , sVotingPeriod :: VotingPeriod
+  }
+  deriving stock (Show)
+
+instance IsoLigo (T.Storage ContractExtraL ProposalMetadataL) Storage where
+  toLigo storage = Storage
+    { sAdmin = T.sAdmin storage
+    , sExtra = T.sExtra storage
+    , sLedger = T.sLedger storage
+    , sMetadata = T.sMetadata storage
+    , sMigrationStatus = T.sMigrationStatus storage
+    , sOperators = T.sOperators storage
+    , sPendingOwner = T.sPendingOwner storage
+    , sPermitsCounter = T.sPermitsCounter storage
+    , sProposals = BigMap . Map.map toLigo . unBigMap $ T.sProposals storage
+    , sProposalKeyListSortByDate = T.sProposalKeyListSortByDate storage
+    , sQuorumThreshold = T.sQuorumThreshold storage
+    , sTokenAddress = T.sTokenAddress storage
+    , sVotingPeriod = T.sVotingPeriod storage
+    }
+  fromLigo Storage{..} = T.Storage
+    { T.sAdmin = sAdmin
+    , T.sExtra = sExtra
+    , T.sLedger = sLedger
+    , T.sMetadata = sMetadata
+    , T.sMigrationStatus = sMigrationStatus
+    , T.sOperators = sOperators
+    , T.sPendingOwner = sPendingOwner
+    , T.sPermitsCounter = sPermitsCounter
+    , T.sProposals = BigMap . Map.map fromLigo $ unBigMap sProposals
+    , T.sProposalKeyListSortByDate = sProposalKeyListSortByDate
+    , T.sQuorumThreshold = sQuorumThreshold
+    , T.sTokenAddress = sTokenAddress
+    , T.sVotingPeriod = sVotingPeriod
+    }
+
+data Config = Config
+  { cDaoName :: MText
+  , cDaoDescription :: MText
```

Comment:
`cDaoName` and `cDaoDescription` are only used in Lorentz autodoc feature. We probably do not need it here.

Reply:
Makes sense to me as well, removed now

Reply:
Related: https://github.com/tqtezos/baseDAO/issues/72